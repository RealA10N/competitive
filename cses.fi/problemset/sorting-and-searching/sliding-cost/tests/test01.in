8 4
2 4 3 5 8 1 2 1
