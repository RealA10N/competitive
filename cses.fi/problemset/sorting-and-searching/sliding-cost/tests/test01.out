4 6 9 10 8
